modal/stub.py

```diff
@@ -479,7 +479,9 @@ def function(
         network_file_systems: Dict[
             Union[str, PurePosixPath], _NetworkFileSystem
         ] = {},  # Mountpoints for Modal NetworkFileSystems
-        volumes: Dict[Union[str, PurePosixPath], _Volume] = {},  # Mountpoints for Modal Volumes
+        volumes: Dict[
```

Comment:
I get why we've reused this, but also worried that doing it this was is more error prone.